app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tab/View.php

```diff
@@ -78,8 +78,12 @@ public function getCustomerLog()
      */
     public function getCreateDate()
     {
-        return $this->_getCoreHelper()->formatDate($this->getCustomer()->getCreatedAt(),
-            Mage_Core_Model_Locale::FORMAT_TYPE_MEDIUM, true);
+        $date = Mage::app()->getLocale()->storeDate(
```

Comment:
Indentation level is off on this whole block.

Reply:
Generally speaking I disagree with this PR since it's not fixing a bug but instead changing behavior to what you expect. This method is meant to return date as it's stored in the DB (which should be UTC I would hope). If you notice, the very next method is `getStoreCreateDate()` and it does exactly what you want. If you think admin should be displaying the store date then instead why not edit the template to use the correct method? Even better would be a new bool config path and then check that in the template to have both behaviors. You could leave the default as the old behavior and use the flag to change to using store times instead.